MobileCenter/MobileCenter/Internals/Device/MSDeviceTracker.m

```diff
@@ -128,7 +128,7 @@ - (MSDevice *)device {
 - (MSDevice *)updatedDevice {
   @synchronized(self) {
     MSDevice *newDevice = [[MSDevice alloc] init];
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_TV
```

Comment:
Do we really need to add this condition? Is possible to have one target that is both iOS and TV OS? It sounds a bit weird :blush: